src/evaluation/policy_evaluation_settings.rs

```diff
@@ -11,4 +11,6 @@ pub(crate) struct PolicyEvaluationSettings {
     pub(crate) allowed_to_mutate: bool,
     /// The policy-specific settings provided by the user
     pub(crate) settings: PolicyOrPolicyGroupSettings,
+    /// Determines a custom error bessage for the policy
+    pub(crate) custom_error_message: Option<String>,
```

Comment:
Should change the name of this field to message? Or change the `message` field in the policies.yml into `customer_error_message`. It like the later because it makes more clear the goal of the configuration.